pkgs/applications/science/biology/sortmerna/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, cmake, rocksdb, rapidjson, pkgconfig, fetchFromGitHub, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "sortmerna";
+  version = "3.0.3";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "biocore";
+    rev = "v${version}";
+    sha256 = "0zx5fbzyr8wdr0zwphp8hhcn1xz43s5lg2ag4py5sv0pv5l1jh76";
+  };
+
+  patches = [ ./CMakeInstallPrefix.patch ];
+
+  nativeBuildInputs = [ cmake rapidjson pkgconfig ];
+  buildInputs = [ zlib rocksdb rapidjson ];
+
+  preConfigure = ''
+    cmakeFlagsArray=(
+      $cmakeFlagsArray
+      "-DCMAKE_INSTALL_PREFIX=$out")
+    '';
+
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DROCKSDB_HOME=${rocksdb}"
+    "-DRAPIDJSON_HOME=${rapidjson}"
+  ];
+
+
+
+  meta = with stdenv.lib; {
+    description = "SortMeRNA is a program tool for filtering, mapping and OTU-picking NGS reads in metatranscriptomic and metagenomic data.";
```

```suggestion
    description = "next-generation sequence filtering and alignment tool";
```

We shouldn't have periods at the end of `description` or restate the package name.
Description is from https://github.com/biocore/sortmerna